less/style.less

```diff
@@ -68,3 +68,8 @@ body {
         line-height: 100px;
     }
 }
+
+.songwriters-details {
+    display: inline-block;
+    text-align: left;
+}
```

Comment:
If we go for the stars design I mentioned in the other PR, you'll need to add the CSS for it. Here's the current one on Sukutomo, which probably needs to be adapted:

```css
.song {
    .level-stars {
	.flaticon-star {
	    &:nth-child(-n+3) { color: #d8fd53; }
	    &:nth-child(n+4):nth-child(-n+6) { color: #f7fb34; }
	    &:nth-child(n+7):nth-child(-n+9) { color: #fbaac3; }
	    &:nth-child(n+10) { color: #f33256; }
	}
    }
}
```

Reply:
I have another PR currently which does something a bit different. I still can go for something like this if you prefer, but I think that it'll look best as just text in the end.